src/app/[locale]/(simulation)/simulateur/[root]/_components/simulateur/form/FunFact.tsx

```diff
@@ -14,28 +14,56 @@ type Props = { question: DottedName }
 export default function FunFact({ question }: Props) {
   const { currentCategory } = useFormState()
 
-  if (question !== 'logement . âge') return null
+  const funFactQuestions: DottedName[] = [
+    'logement . âge',
+    'divers . textile . volume',
+  ]
 
-  return (
-    <div
-      className={`rounded-xl border-2 ${getBorderCategoryColor(currentCategory, '200')} ${getBgCategoryColor(currentCategory, '100')} px-4 py-6 text-sm`}>
-      <h3 className="mb-2">
-        <Emoji>💡</Emoji> Le saviez vous ?
-      </h3>
-      <p className="mb-0">
-        <strong>
-          <Trans>
-            La taille des logements français a très fortement augmenté
-          </Trans>
-        </strong>{' '}
-        <Trans>
-          sur ces 50 dernières années, passant de 23 à 40,4 m² par habitant,
-          soit
-        </Trans>{' '}
-        <strong>
-          <Trans>90,9 m² en moyenne pour un foyer de 2,2 personnes.</Trans>
-        </strong>
-      </p>
-    </div>
-  )
+  if (funFactQuestions.includes(question)) {
```

Comment:
Il vaut mieux éviter les if imbriqués pour faciliter la lecture ; aussi je trouve ça dommage d'écrire en dur à trois endroits les noms de règles, pour y remédier je te propose : 

```suggestion
// Logement âge question
  if (funFactQuestions.indexof(question) === 0) {
  // ici tu retournes ton bloc 
  }
  // Divers textile volume question
  if (funFactQuestions.indexof(question) === 1) {
  // ici tu retournes ton bloc
  }
  return null
```

Reply:
Je préfère pas utiliser l'indexation des array car je pense que c'est pas fiable ..

Reply:
c'est peut-être indexOf

Reply:
Je suis pas très à l'aise avec l'idée de se fier aux index des array

Reply:
Je me souviens avoir eu des bugs entre différents navigateurs (genre les mosaiques)

Reply:
Quand les arrays sont dynamiques potentiellement ça peut poser problème mais pas là il me semble 🤔